awx/sso/ldap_group_types.py

```diff
@@ -13,7 +13,7 @@
 
 class PosixUIDGroupType(LDAPGroupType):
 
-    def __init__(self, ldap_group_user_attr, *args, **kwargs):
+    def __init__(self, name_attr='cn', ldap_group_user_attr='uid', *args, **kwargs):
         super(PosixUIDGroupType, self).__init__(*args, **kwargs)
```

Comment:
This won't pass `name_attr` to the superclass.

Reply:
indeed